dashboard/src/components/Accordion/BuildAccordionContent.tsx

```diff
@@ -1,10 +1,10 @@
 import { MdFolderOpen } from 'react-icons/md';
 
-import { useMemo, useCallback } from 'react';
+import { useCallback, useMemo } from 'react';
 
 import { FormattedMessage } from 'react-intl';
 
-import { useNavigate, useParams } from '@tanstack/react-router';
+import { useParams, useNavigate } from '@tanstack/react-router';
```

Comment:
is there ar eason for the changes in this file? it seems a little like bit like noise

Reply:
I agree, the linter doesn't seem to bother with the order of this import